### PR TITLE
masking liscence plate

### PR DESCRIPTION
**Pull request by doorooful**
- Liscense plate detection with pretrained model(roboflow)
- Masking detected plate with (128, 128, 130) color
- Updated README for demo app

### CODE REVIEW

**Comment by doorooful** [comment]
- Result when image uploaded

image


https://github.com/jeewonkimm2/generative-inpainting-pytorch/assets/110809138/84b7408d-966b-4240-b904-950244870225
